docs/Atari-800-and-5200.md

```diff
@@ -62,6 +62,14 @@ atari800_resolution = "336x240"
 atari800_sioaccel = "enabled"
 atari800_system = "5200"
 ```
+
+Edit `/opt/retropie/configs/atari5200/emulators.cfg` and replace `retroarch.cfg` with `retroarch-core-options.cfg`.
```

Comment:
I don't think that's right - that's the retroarch cfg append file, for settings, not core options. i'm surprised it works, but you probably don't want to lose the normal functionality.  would it work instead to set
```global_core_options = "false"```
 in `/opt/retropie/configs/atari5200/retroarch.cfg` and then i think it will pick up the retroarch-core-options.cfg file automatically (?)

Reply:
The underlying mechanism for placing retroarch-core-options.cfg into the atari5200 directory is a black box for me at this time. It seemed odd that it was not referenced in the 800/5200 docs, and it has not been picked up automatically by any normal method I have tried.

I should have been more explicit as to what I edited in the emulators.cfg ... it is the --config param, the original is:
`lr-atari800 = "/opt/retropie/emulators/retroarch/bin/retroarch -L /opt/retropie/libretrocores/lr-atari800/atari800_libretro.so --config /opt/retropie/configs/atari5200/retroarch.cfg %ROM%"`

To try this out I:
- Deleted my working retroarch-core-options.cfg file and replaced it with the prescribed 10 lines from the existing doc.
- Edited emulators.cfg (--config param) to use retroarch.cfg
- retroarch.cfg has (2) uncommented lines - I added the global_core_options you suggested above:
`input_remapping_directory = "/opt/retropie/configs/atari5200/"
global_core_options = "false"`

When I launch Berzerk from EmulationStation I am returned back to ES - no Memo Pad.
I also tried a full system reboot ... no change.
/dev/shm/runcommand.log reports a segmentation fault:
```/opt/retropie/supplementary/runcommand/runcommand.sh: line 1283:   999 Segmentation fault      /opt/retropie/emulators/retroarch/bin/retroarch -L /opt/retropie/libretrocores/lr-atari800/atari800_libretro.so --config /opt/retropie/configs/atari5200/retroarch.cfg "/home/pi/RetroPie/roms/atari5200/Berzerk (USA).zip" --appendconfig /dev/shm/retroarch.cfg```

Afterward, I edited emulators.cfg and again changed the --config param to point to the 10-line retroarch-core-options.cfg and I'm able to launch any game I choose without issues.  Also, the core options within the 5200 path has been fully populated.

/dev/shm/runcommand.log (after):
```$ cat /dev/shm/runcommand.log 
Parameters: 
Executing: /opt/retropie/emulators/retroarch/bin/retroarch -L /opt/retropie/libretrocores/lr-atari800/atari800_libretro.so --config /opt/retropie/configs/atari5200/retroarch-core-options.cfg "/home/pi/RetroPie/roms/atari5200/Berzerk (USA).zip" --appendconfig /dev/shm/retroarch.cfg
[libretro-atari800]: Got size: 336 x 240.
[libretro-atari800]: Got size: 336 x 240.
[libretro-atari800]: Got size: 336 x 240.
[libretro-atari800]: Got size: 336 x 240.```

I'm still a n00b, but learning.